redfish-core/lib/led.hpp

```diff
@@ -323,6 +323,23 @@ inline void
         std::variant<bool>(ledState));
 }
 
+inline bool validateLedState(const std::shared_ptr<bmcweb::AsyncResp>& aResp,
+                             const std::string& ledState)
+{
+    if (ledState.empty())
+    {
+        return true;
```

Comment:
Why return true here? Shouldn't this be false?

Reply:
No.
ledState equal to empty means no need to process ledState attributes.

Reply:
but below I see 

    if (!validateLedState(aResp, ledState))
    {
        return;
    }

Reply:
as above